README.md

```diff
@@ -1,70 +1,90 @@
-| [OneSignal Expo EAS Plugin](https://github.com/OneSignal/onesignal-expo-plugin)                   | [OneSignal Expo EAS Plugin Setup](https://documentation.onesignal.com/docs/react-native-expo-sdk-setup)   |                                                                                                                              |
-| [OneSignal Gradle Plugin](https://github.com/OneSignal/OneSignal-Gradle-Plugin)                   |                                                                                                           |                                                                                                                        |
-| [OneSignal CLI](https://github.com/OneSignal/onesignal-cli)                                       |                                                                                                           |                                                                                                                        |
+| Platform                                                                                        | Getting Started Guide                                                                                   | SDK/Plugin Listing |
+| ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | ------------------ |
+| [OneSignal Android Studio Plugin](https://github.com/OneSignal/onesignal-android-studio-plugin) |                                                                                                         |                    |
+| [OneSignal Expo EAS Plugin](https://github.com/OneSignal/onesignal-expo-plugin)                 | [OneSignal Expo EAS Plugin Setup](https://documentation.onesignal.com/docs/react-native-expo-sdk-setup) |                    |
+| [OneSignal Gradle Plugin](https://github.com/OneSignal/OneSignal-Gradle-Plugin)                 |                                                                                                         |                    |
+| [OneSignal CLI](https://github.com/OneSignal/onesignal-cli)                                     |                                                                                                         |                    |
+
+# Deprecated
+
+| Platform                                                                          | Date               |
+| :-------------------------------------------------------------------------------- | ------------------ |
+| [OneSignal Alexa Plugin](https://github.com/OneSignal/OneSignal-Alexa-Nodejs-SDK) | <INSERT DATE HERE> |
```

Comment:
@iAmWillShepherd Should there be a date here?

Reply:
The date is for your team to decide. I'm not sure when/if y'all have officially deprecated it yet.